fuzz/tls_init/fuzz_init.cc

```diff
@@ -0,0 +1,79 @@
+/* Copyright (c) 2021 Intel Corporation
+ * Copyright (c) 2020-2021 Alibaba Cloud
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+extern "C"{
+#include <stdint.h>
+#include <stddef.h>
+#include <string.h>
+#include "rats-tls/api.h"
+#include "rats-tls/log.h"
+#include "rats-tls/claim.h"
+#include "internal/core.h"
+}
+#include <fuzzer/FuzzedDataProvider.h>
+#include <vector>
+
+#define CUSTOM_CLAIMS_SIZE 10
+using namespace std;
+
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t *data,size_t size){
+    rats_tls_conf_t conf; // consume 192 bytes
+    // conf | claim_array | random char *  in claim_array
+    if(size < sizeof(rats_tls_conf_t) + 10 * sizeof(claim_t) + 50 * 10){
+        return 0;
+    }
+    memcpy(&conf, data, sizeof(conf));
+    conf.log_level  = RATS_TLS_LOG_LEVEL_DEFAULT;
+    conf.api_version = 0;
+    
+    /*fuzz log level*/
+    /*fuzz round could not be too huge, that leads to unexpected log_level*/
+    strcpy(conf.attester_type, "nullattester");
+    strcpy(conf.verifier_type, "nullverifier");
+    strcpy(conf.tls_type, "nulltls");
```

Comment:
Does other fileds in rats_tls_conf_t such as `attester_type ` or `verifier_type ` etc need to be fuzzed?

Reply:
You mean fill them totally  with random string or different type like "sgx_la" .
And for host mode, I also should use different attester/verifier type too?

Reply:
yes. In host mode, there are different attester/verifier type. We cannot assume that we are using the null type, the user may output different types, and the program will output different error logs.